CHANGES.md

```diff
@@ -12,3 +12,14 @@ rsh46@cam.ac.uk
 |---------------|---------------|-----------------|
 | index.php     |            48 | File space reporting using df                        |
 |               |           159 | Sort datasets by descending name CTR0001 compatible  |
+
+
+
+
+#### Git Submodules ####
+
+# To read Excel files and display in a very simple html format
+git submodule add https://github.com/PHPOffice/PHPExcel PHPExcel
```

Comment:
You don't really need to put this in any docs - you've run the command once and now it's saved in the repo, so anyone else who downloads it should get it without running anything themselves.

I think.